niet/__main__.py

```diff
@@ -6,10 +6,15 @@
 import yaml
 
 
+# arguments parsing
 def argparser():
     epilog = '''output formats:
-             quotes    Add quotes to result
-             '''
+  spaces    Return all elements of a list separated by spaces
+  quotes    Add quotes to result
+  newline   Return all element of a list in a new line
+  yaml      Return object in YAML
+  json      Return object in JSON
+'''
```

Comment:
First you need to import futur:
```python
from future.utils import viewitems
```
Now you can deal with dict and python2/3 in the same way:
```python
epilog = ''' output formats:\n{}'''.format("\n".join(["{} {}".format(key, value["epilog"]) for key, value in viewitems(VALID_PRINTERS)]))
```